generators/entity-server/templates/src/main/java/package/domain/Entity.java.jhi.spring_data_cassandra.ejs

```diff
@@ -30,7 +30,9 @@ import org.springframework.data.cassandra.core.mapping.Table;
 <&_ } -&>
 
 <&_ if (fragment.annotationSection) { -&>
-@Table("<%= entityInstance %>")
+// Since spring-data-cassandra 3.4.2, table names needs to be in lowercase
+// See https://github.com/spring-projects/spring-data-cassandra/issues/1293#issuecomment-1192555467
+@Table("<%= entityInstance.toLowerCase() %>")
```

Comment:
@mshima Not sure it deserve any pre-built table name variable dependent on the database?

Reply:
@DanielFran I think we should do it.
This will make easier when customizing using blueprints and jdl.
And allows to keep synced with changelogs:
https://github.com/jhipster/generator-jhipster/blob/11638c11e5d45b8817fb1324e388e20486808e35/generators/entity-server/templates/src/main/resources/config/cql/changelog/added_entity.cql.ejs#L20